client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/default-contact.tsx

```diff
@@ -0,0 +1,74 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
This file [looks pretty close to this one](https://github.com/Automattic/wp-calypso/blob/update/plans-congrats/client/my-sites/checkout/checkout-thank-you/marketplace/masterbar-styled/default-contact.tsx). Do we have a special reason to not reuse/extend that one, or is the goal here to isolate everything by rewriting the components?

Reply:
This will be the main component that we will use. it will be that the marketplace one (old one) should link to this one in another PR.